src/xcomponentAPI.ts

```diff
@@ -23,12 +23,12 @@ class XComponentAPI {
         }).bind(this));
     };
 
-    createSession(xcApiFileName: string, serverUrl: string, createSessionListener: (error: Error, session: Session) => void, reconnectionAfterError: boolean = false, reconnectionIntervalSeconds: number = 10): void {
-        this.connection.createSession(xcApiFileName, serverUrl, createSessionListener, reconnectionAfterError, reconnectionIntervalSeconds);
+    createSession(xcApiFileName: string, serverUrl: string, createSessionListener: (error: Error, session: Session) => void, deconnectionErrorListener?: (closeEvent: CloseEvent) => void): void {
```

Comment:
Ok. got it. Looks fine.
On the PR with the retry policies, we'd need an extra `options` parameter here, with the reconnection parameters. Just like in the slack API.